src/tools/pbs_probe.c

```diff
@@ -2244,7 +2244,7 @@ pbsdotconf(char * path)
 static	int
 get_realpath_values(struct infrastruct *pinf)
 {
-	char real[MAXPATHLEN];
+	char real[PATH_MAX];
```

Comment:
Now that you've fixed the realpath function for Windows, this should be changed to:
`
char *real = NULL;
`
The realpath() calls should be passed NULL as the second argument, and the return value assigned to real. Don't forget to check the value after the call, and don't forget to free it when you're done.

Reply:
Yes will do in another commit. I am trying to logically separate the changes to multiple commits.

Reply:
Fair enough. Wasn't certain how you intended to proceed.

Reply:
Sorry for the confusion. I will remove the WIP tag once I am all done. Then you can review all changes at once.